content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -418,6 +418,20 @@ Used on: Secret
 This annotation records the {{< glossary_tooltip term_id="uid" text="unique ID" >}} of the
 ServiceAccount that the token (stored in the Secret of type `kubernetes.io/service-account-token`) represents.
 
+### kubernetes.io/legacy-token-last-used
+
+Example: `kubernetes.io/legacy-token-last-used: 2022-10-24`
+
+Used on: Secret
+
```

Comment:
@sftim  should this note the feature state or the feature-gate required to be enabled for this to be set?

Reply:
It's a nice to have, sure. We can merge this change without that detail.